environments/ithc/ithc.tfvars

```diff
@@ -478,6 +478,12 @@ frontends = [
     backend_domain = ["firewall-nonprodi-palo-ithc.uksouth.cloudapp.azure.com"]
 
   },
+  {
+    name           = "lau-case"
+    mode           = "Detection"
+    custom_domain  = "lau-case.ithc.platform.hmcts.net"
```

Comment:
I had a quick look and couldn't see the required dns records, can you check these are there first please?
https://hmcts.github.io/ways-of-working/path-to-live/public-dns.html#setup-required-for-an-application

Reply:
Sorry, only AAT was added. I've added ITHC and Perftest in https://github.com/hmcts/azure-public-dns/pull/659